Add referral reward docs (#31)
paragraph-xyz/docs: src/pages/docs/advanced/referral-program.md

---
title: Referral rewards
description: Incentivize & reward readers, while ensuring curators and your audience gets paid for driving traffic.
---

Every Paragraph publication has a referral program built-in, to help creators grow by incentivizing & rewarding readers for driving traffic.

Simply append a wallet address to any Paragraph URL, and when users click on the link, you'll get a reward fee automatically sent to your wallet whenever users mint collectible posts.

> If a user clicks your referral link then becomes a Paragraph creator themself, you'll automatically get an indefinite

Reviewer: ```suggestion
If a user clicks your referral link and then becomes a Paragraph creator themselves, you'll automatically get an indefinite
```
I'm not sure themself is a word. I think it would be themselves - at least I've never seen themself before.

Reviewer 2: Wow this led me down a little interesting rabbit hole. TLDR: both are accepted but themselves is more common even for singular.

https://apastyle.apa.org/style-grammar-guidelines/grammar/singular-they#:~:text=Use%20a%20singular%20verb%20form,currently%20the%20more%20common%20usage.

> rake of all revenue they earn on Paragraph.

Reviewer: ```suggestion
rake of all the revenue they earn on Paragraph.
```

- A newsletter: https://paragraph.xyz/@blog?referrer=0x123
- Any other page: https://paragraph.xyz?referrer=0x123

Whenever your audience clicks on that link, we set your wallet address as the 'referer' for that user
for the next 7 days.

If the user mints an NFT on the newsletter you send them to, you get the 'reader referrer' fee.

If the user registers a _new_ account on Paragraph and begins creating content themselves, you'll
indefinitely be set as the 'creator referrer' for them, and you'll get the 'creator referrer' fee
every time they monetize.

## Reward Fees

It's free and gasless to create collectible posts on Paragraph. We charge a fixed fee for all collectors.
If there are no referrers for the given transaction, Paragraph takes the full fee from the referrer addresses.

### Optimism & Base
We charge a 0.000777 ETH mint fee, with the following reward breakdown:

- 0.000333 ETH to the creator
- 0.000222 ETH to the Paragraph platform
- 0.000111 ETH to the 'creator referrer'
- 0.000111 ETH to the 'reader referrer'

### Polygon
We charge a 2 MATIC mint fee, with the following reward breakdown:

- 0.8 MATIC to the creator
- 0.6 MATIC to the Paragraph platform

> - 0.3 MATIC tot he creator referrer

Reviewer: ```suggestion
- 0.3 MATIC to the creator referrer
```
